apps/demo-game/src/pages/admin/games.tsx

```diff
@@ -94,14 +94,29 @@ function Games() {
         )}
       </Formik>
 
-      <div className="mt-4 grid grid-cols-10 gap-1">
-        {data.games.map((game) => (
-          <div className="border p-2" key={game?.id}>
-            <Link href={`/admin/games/${game?.id}`}>
-              {game?.name} {game?.activePeriodIx}
-            </Link>
-          </div>
-        ))}
+      <div className="mt-4 flex flex-col gap-1">
+        {[...data.games]
```

Comment:
do the ordering in prisma using orderBy instead of on the frontend

Reply:
Ah very true! Thanks!
It helped me to check how the useQuery works and which query function is called on the back-end.